datahub-web-react/src/components/Routes.tsx

```diff
@@ -27,18 +28,28 @@ const ProtectedRoute = ({
  */
 export const Routes = (): JSX.Element => {
     const isLoggedIn = useReactiveVar(isLoggedInVar);
-
     return (
         <div>
             <Switch>
+                <ProtectedRoute isLoggedIn={isLoggedIn} exact path="/" render={() => <BrowseTypesPage />} />
                 <Route path={PageRoutes.LOG_IN} component={LogIn} />
                 <ProtectedRoute
                     isLoggedIn={isLoggedIn}
                     path={`${PageRoutes.DATASETS}/:urn`}
                     render={() => <DatasetPage />}
                 />
                 <ProtectedRoute isLoggedIn={isLoggedIn} path={PageRoutes.SEARCH} render={() => <SearchPage />} />
-                <ProtectedRoute isLoggedIn={isLoggedIn} path={PageRoutes.BROWSE} render={() => BrowsePage} />
+                <ProtectedRoute
+                    isLoggedIn={isLoggedIn}
+                    exact
+                    path={PageRoutes.BROWSE}
```

Comment:
One nit: PageRoutes.BROWSE -> BrowseTypesPage and PageRoutes.BROWSE_TYPE -> BrowseResultsPage seems a bit confusing.. Can we making the naming more consistent here?

Maybe we can rename the PageRoutes as BROWSE_TYPES and BROWSE_RESULTS, respectively. Thoughts?

Reply:
great idea